linear_algebra/multivariate_polynomial.lean

```diff
@@ -266,6 +266,40 @@ instance map.is_ring_hom [decidable_eq β] [comm_ring β]
   is_ring_hom (map f : mv_polynomial σ α → mv_polynomial σ β) :=
 map₂.is_ring_hom _ _
 
+variables {p q : mv_polynomial σ α}
+
+section map₂
+
+variables [comm_ring β] [decidable_eq β]
+variables (f : σ → β) (g : α → β) [is_ring_hom g]
+
+@[simp] lemma map₂_sub : (p - q).map₂ f g = p.map₂ f g - q.map₂ f g := is_ring_hom.map_sub _
```

Comment:
this and the other `sub` lemmas should not be `[simp]`

Reply:
Why not? What is the difference between an `add` lemma and a `sub` lemma? Is it that `a - b` is complicated, and you prefer `a + -b`?

Reply:
it's not so much that I prefer it, but `a - b = a + -b` is a simp lemma whether I like it or not. It will still simplify, it will just use the other lemmas to do so.

Reply:
Done